packages/features/insights/server/trpc-router.ts

```diff
@@ -90,6 +90,7 @@ const UserSelect = {
   name: true,
   email: true,
   avatar: true,
```

Comment:
@alannnc is this correct usage of avatar. i think fetching avatar caused some issues in event-types.